pages/18-01.md

```diff
@@ -48,6 +48,9 @@ Based on current network scan data and a clear potential for harm, this
 directive requires actions related to two topics: email security and web
 security.
 
+Based on discussions with Federal agency leaders and industry representatives, DHS acknowledges that there is a significant constraint around disabling the weak email cipher 3DES. As a result, DHS is hereby issuing a [temporary policy exception on the BOD requirement related to disabling the weak email cipher 3DES.](https://cyber.dhs.gov/assets/report/<BOD_18-01_Letter_for_Temporary_Exception_to_Agencies_FINAL_v1_Signed.pdf)
```

Comment:
Is the `<` expected in the URL/filename? And for this to be a 404?

Reply:
Thanks @konklone.